ethereum_history_api/oracles/src/noir/oracles/accountOracle.ts

```diff
@@ -1,23 +1,11 @@
 import { type ForeignCallOutput } from '@noir-lang/noir_js';
-import { fromRlp, type GetProofReturnType, type Hex, isHex, type PublicClient } from 'viem';
+import { type Hex, isHex, type PublicClient } from 'viem';
 import { assert } from '../../util/assert.js';
-import {
-  ADDRESS_LENGTH,
-  PROOF_ONE_LEVEL_LENGTH,
-  ZERO_PAD_VALUE,
-  decodeField,
-  decodeHexAddress,
-  encodeField,
-  encodeHex,
-  encodeProof
-} from './encode.js';
-import { padArray } from '../../util/array.js';
 import { NoirArguments } from './oracles.js';
+import { ADDRESS_LENGTH } from './codec/const.js';
+import { decodeField, decodeHexAddress } from './codec/decode.js';
+import { encodeAccount, encodeStateProof } from './accountOracle/codec.js';
```

Comment:
In this file we could rename 
`parseNoirGetAccountArguments`
to 
`decodeGetAccountArguments`
for naming consistency